indico/htdocs/sass/partials/_icons.scss

```diff
@@ -94,445 +94,448 @@ i[class^="icon-"]:before {
 
 /* Icomoon's icon definitions */
 .icon-agreement:before {
-    content: "\e605";
+  content: "\e605";
```

Comment:
select the whole generated block and let pycharm reformat it to keep the 4-space indentation